### PR TITLE
Register teamine.is-a.dev

### PR DESCRIPTION
**Pull request by WellandDev**
Register teamine.is-a.dev with А

### CODE REVIEW

File: domains/teamine.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "WellandDev",
+           "email": "rataretimig1502@gmail.com",
+           "discord": "905515659524472952"
+        },
+    
+        "record": {
+            "TXT": "domain for minecraft server dev."
```
Comment (DEV-DIBSTER) [inline domains/teamine.json:9]: TXT record alone does nothing.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/teamine.json (hunk 1 above)
Comment (WellandDev) [reply domains/teamine.json:9]: Please forgive me, I was reading the brief documentation and rushed into action. 


Is it possible to re-register the subdomain?

Comment (WellandDev) [reply domains/teamine.json:9]: This is my first time working with domain names and I may be confused. I apologize in advance

**Review by triach-rold — CHANGES_REQUESTED** [review]
You need to have a valid CNAME pointing to a website that is reachable and has enough content on it to be considered as complete.

File: domains/teamine.json (hunk 1 above)
Comment (21Z) [reply domains/teamine.json:9]: You can update the domain file on the fork of this repo on your account, and the changes will automatically be added to this PR

**Comment by CuteDog5695** [comment]
> You need to have a valid CNAME pointing to a website that is reachable and has enough content on it to be considered as complete.

not everything needs to be CNAME btw

**Comment by orxngc** [comment]
Stale PR, closing

**Pull request closed without merging**